File: r/add.r
add <- function(firstNumber, secondNumber){
	sum <- firstNumber + secondNumber
	 return ( sum )

}

firstNumber <- readline(prompt = "Enter first number: ")
secondNumber <- readline(prompt = "Enter second number: ")
sum <- add(firstNumber, secondNumber)
print("Sum of ", firstNumber, ", ", secondNumber, " is ", sum, ".")

File: perl/add.pl
sub add{
	($firstNumber, $secondNumber) = @_;  
	$sum = $firstNumber + $secondNumber;
	return $sum;
}


print "Enter first number: ";
$firstNumber = 3; #<STDIN>;
print "Enter second number: ";
$secondNumber = <STDIN>;
$sum = add($firstNumber , $secondNumber);
print "Sum of $firstNumber and $secondNumber is $sum."
